docs/sampling.asciidoc

```diff
@@ -1,27 +1,77 @@
 [[sampling]]
 === Transaction sampling
 
-Elastic APM supports head-based, probability sampling.
-_Head-based_ means the sampling decision for each trace is made when that trace is initiated.
-_Probability sampling_ means that each trace has a defined and equal probability of being sampled.
+Distributed tracing can generate a substantial amount of data.
+More data can mean higher costs and more noise to sift through.
+Sampling aims to lower the amount of data ingested and the effort required to analyze that data --
+all while still making it easy to find anomalous patterns in your applications, detect outages, track errors,
+and lower MTTR.
+
+Elastic APM supports two types of sampling:
+
+* <<head-based-sampling>>
+* <<tail-based-sampling>>
+
+[float]
+[[head-based-sampling]]
+==== Head-based sampling
+
+In head-based sampling, the sampling decision for each trace is made when that trace is initiated.
+Each trace has a defined and equal probability of being sampled.
 
 For example, a sampling value of `.2` indicates a transaction sample rate of `20%`.
 This means that only `20%` of traces will send and retain all of their associated information.
 The remaining traces will drop contextual information to reduce the transfer and storage size of the trace.
 
+Head-based sampling is quick and easy to set up.
+Its downside is that it's entirely random -- good
+data might be discarded purely due to chance.
+
+See <<configure-head-based-sampling>> to get started.
+
+**Distributed tracing with head-based sampling**
+
+In a distributed trace, the sampling decision is still made when the trace is initiated.
+Each subsequent service respects the initial service's sampling decision, regardless of its configured sample rate;
+the result is a sampling percentage that matches the initiating service.
+
+In this example, `Service A` initiates four transactions and has sample rate of `.5` (`50%`).
+The sample rates of `Service B` and `Service C` are ignored.
+
+image::./images/dt-sampling-example-1.png[Distributed tracing and head based sampling example one]
+
+In this example, `Service A` initiates four transactions and has a sample rate of `1` (`100%`).
+Again, the sample rates of `Service B` and `Service C` are ignored.
+
+image::./images/dt-sampling-example-2.png[Distributed tracing and head based sampling example two]
+
 [float]
-==== Why sample?
+[[tail-based-sampling]]
+==== Tail-based sampling
```

Comment:
Should we also mention the fact that tail-based sampling uses weighted random sampling? We weight the random sampling of traces based on the root transaction duration, which has the effect of preferring to keep slower traces rather than faster ones.

Reply:
Yes! I wasn't aware of this. I'll add it in.